apps/portal/README.md

```diff
@@ -1,4 +1,4 @@
-# OsmoNotifyFrontend
+# OsmoXFrontend
```

Comment:
Update to OsmoX Portal, update the package.json and angular.json too, see if it takes more time, if yes do as different task instead.

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b